### ISSUE TITLE
Automate release process

### ISSUE THREAD

**Issue by netbe** [OpenShelter/fastlane-plugin-rome#4]
* [ ] define what to do exactly on that issue
* [ ] add CI checks / tests (if needed)
* [ ] add CI build on release tag (is it the right way🤷‍♂️)

**Comment by tmspzz** [comment]
Can you elaborate more on `define what to do exactly on that issue` ? What do you mean?

Thanks for moving the repo here btw ❤️

**Comment by netbe** [comment]
@blender yeah of course.

I thought it was better to move it here instead of adding you as collaborator on that repo.

My first thoughts:

Scenario:

- one makes a change with a PR
- tests are run on PR
- once reviewed and tests passed, one can merge the PR
- To release, one can go to release https://github.com/OpenShelter/fastlane-plugin-rome/releases, draft the release. This will tag the version (this means the version will be changed beforehand in code, or is there another way to do it?)
- The CI sees the tag and proceeds with release doing a `rake release` to *rubygems*.

**Comment by tmspzz** [comment]
So basically we only need to setup travis, I'm I right?

Afaik you have to change the version for the release manually prior to running `rake release`

**Comment by netbe** [comment]
yes that's right. That's the setup I have on my other repo, maybe this will suffice: 
https://github.com/netbe/Babelish/blob/master/.travis.yml

**Comment by tmspzz** [comment]
I think it will.

As for the automated release I don't think that's possible. I did some googling but didn't find much.
